backend/gateway/meta/meta.go

```diff
@@ -171,3 +173,38 @@ func resolvePattern(message descriptor.Message, pattern *apiv1.Pattern) *auditv1
 
 	return &auditv1.Resource{TypeUrl: pattern.TypeUrl, Id: resourceName}
 }
+
+// isValidInterface will return true if the type is proto.message and value is not nil
+func isValidInterface(body interface{}) bool {
```

Comment:
I took this approach because we know what type we want to process (proto message) and know we don't want to process a proto message with a nil value. A scenario where `body == nil` or is any other type than proto.message, would fall to the default cause.